packages/packages.csv

```diff
@@ -58,9 +59,6 @@ Microsoft.AspNetCore.HttpOverrides,ship
 Microsoft.AspNetCore.Identity,ship
 Microsoft.AspNetCore.Identity.EntityFrameworkCore,ship
 Microsoft.AspNetCore.IISPlatformHandler,noship
-Microsoft.AspNetCore.Internal.libuv-Darwin,noship
-Microsoft.AspNetCore.Internal.libuv-Linux,ship
-Microsoft.AspNetCore.Internal.libuv-Windows,noship
```

Comment:
Spoke to @halter73 offline. I'll leave these as is and mark them as `noship`. He'll make a change to `libuv-package` to copy the nupkgs to the artifacts directory at which point we'll redo - https://github.com/aspnet/Coherence/pull/71/files#diff-941327d979d4d3fb4ad5068c28050da3L21